karma.conf.js

```diff
@@ -14,13 +14,13 @@ module.exports = function(config) {
 
     // list of files / patterns to load in the browser
     files: [
-        'bower_components/angular/angular.js',
-        'bower_components/angular-route/angular-route.js',
-        'bower_components/angular-sanitize/angular-sanitize.js',
-        'bower_components/google-code-prettify/src/prettify.js',
-        'node_modules/angular-mocks/angular-mocks.js',
-        './app/**/*.js',
-        './tests/**/*Spec.js'
+        './coalahtml/_coalahtml/bower_components/angular/angular.js',
+        './coalahtml/_coalahtml/bower_components/angular-route/angular-route.js',
+        './coalahtml/_coalahtml/bower_components/angular-sanitize/angular-sanitize.js',
+        './coalahtml/_coalahtml/bower_components/google-code-prettify/src/prettify.js',
+        './node_modules/angular-mocks/angular-mocks.js',
+        './coalahtml/_coalahtml/app/**/*.js',
+        'tests/**/*Spec.js'
```

Comment:
Forgot to add that. Will add:+1:
ᐧ

On Thu, May 12, 2016 at 11:57 PM, Attila <notifications@github.com> wrote:

> In karma.conf.js
> <https://github.com/coala-analyzer/coala-html/pull/45#discussion_r63071928>
> :
>
> > @@ -14,13 +14,13 @@ module.exports = function(config) {
> >
> >      // list of files / patterns to load in the browser
> >      files: [
> > -        'bower_components/angular/angular.js',
> > -        'bower_components/angular-route/angular-route.js',
> > -        'bower_components/angular-sanitize/angular-sanitize.js',
> > -        'bower_components/google-code-prettify/src/prettify.js',
> > -        'node_modules/angular-mocks/angular-mocks.js',
> > -        './app/**/*.js',
> > -        './tests/**/*Spec.js'
> > +        './coalahtml/_coalahtml/bower_components/angular/angular.js',
> > +        './coalahtml/_coalahtml/bower_components/angular-route/angular-route.js',
> > +        './coalahtml/_coalahtml/bower_components/angular-sanitize/angular-sanitize.js',
> > +        './coalahtml/_coalahtml/bower_components/google-code-prettify/src/prettify.js',
> > +        './node_modules/angular-mocks/angular-mocks.js',
> > +        './coalahtml/_coalahtml/app/**/*.js',
> > +        'tests/**/*Spec.js'
>
> Why you add everywhere ./ and here remove it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-html/pull/45/files/488e627e81af4f5ed6577b02897b51388ccabc49#r63071928>
>